build/Targets/Settings.targets

```diff
@@ -78,6 +78,7 @@
 
   <PropertyGroup Condition="'$(DevEnvDir)' == ''">
     <DevEnvDir>$(VSINSTALLDIR)\Common7\IDE</DevEnvDir>
+    <DevEnvDir Condition="'$(TF_BUILD_BUILDNUMBER)' != ''">$(VS140COMNTOOLS)\..\IDE</DevEnvDir>
```

Comment:
Fancy magic where we construct a `VSCOMNTOOLS` variable that is set to the value of `VS$(VisualStudioReferenceMajorVersion)COMNTOOLS`